src/components/utils/detectMobileOS.js

```diff
@@ -0,0 +1,9 @@
+export const getMobileOSName = () => {
+  const userAgent = navigator.userAgent
```

Comment:
Still not too keen on using `navigator.userAgent`. Is this a permanent solution or are we just using temporarily? What's the reliability `navigator.userAgent`? Did you guys discussed any possible alternative?

Reply:
I think it's fine to use this function in this case because we are just returning a string but it would be good to add a comment to prevent people to start using it.

Reply:
Without any user agent detection, the trade off would be a longer message to cover both Androids & iOS though we're trying to keep screens less wordy.

Will add the note to discourage usage of it.